src/modules/navigator/navigator.h

```diff
@@ -133,20 +134,21 @@ class Navigator : public control::SuperBlock
 	/**
 	 * Getters
 	 */
-	struct vehicle_status_s	    *get_vstatus() { return &_vstatus; }
-	struct vehicle_land_detected_s	    *get_land_detected() { return &_land_detected; }
-	struct vehicle_control_mode_s	    *get_control_mode() { return &_control_mode; }
-	struct vehicle_global_position_s   *get_global_position() { return &_global_pos; }
-	struct vehicle_gps_position_s	    *get_gps_position() { return &_gps_pos; }
-	struct sensor_combined_s	    *get_sensor_combined() { return &_sensor_combined; }
-	struct home_position_s		    *get_home_position() { return &_home_pos; }
-	bool				    home_position_valid() { return (_home_pos.timestamp > 0); }
-	struct position_setpoint_triplet_s *get_position_setpoint_triplet() { return &_pos_sp_triplet; }
-	struct position_setpoint_triplet_s *get_reposition_triplet() { return &_reposition_triplet; }
-	struct position_setpoint_triplet_s *get_takeoff_triplet() { return &_takeoff_triplet; }
-	struct mission_result_s	    *get_mission_result() { return &_mission_result; }
-	struct geofence_result_s		    *get_geofence_result() { return &_geofence_result; }
-	struct vehicle_attitude_setpoint_s *get_att_sp() { return &_att_sp; }
+	struct vehicle_status_s							*get_vstatus() { return &_vstatus; }
+	struct vehicle_land_detected_s			*get_land_detected() { return &_land_detected; }
+	struct vehicle_control_mode_s				*get_control_mode() { return &_control_mode; }
+	struct vehicle_global_position_s		*get_global_position() { return &_global_pos; }
+	struct vehicle_local_position_s			*get_local_position() { return &_local_pos; }
+	struct vehicle_gps_position_s				*get_gps_position() { return &_gps_pos; }
+	struct sensor_combined_s						*get_sensor_combined() { return &_sensor_combined; }
+	struct home_position_s							*get_home_position() { return &_home_pos; }
+	bool home_position_valid() { return (_home_pos.timestamp > 0); }
+	struct position_setpoint_triplet_s 	*get_position_setpoint_triplet() { return &_pos_sp_triplet; }
+	struct position_setpoint_triplet_s 	*get_reposition_triplet() { return &_reposition_triplet; }
+	struct position_setpoint_triplet_s 	*get_takeoff_triplet() { return &_takeoff_triplet; }
+	struct mission_result_s							*get_mission_result() { return &_mission_result; }
+	struct geofence_result_s						*get_geofence_result() { return &_geofence_result; }
+	struct vehicle_attitude_setpoint_s 	*get_att_sp() { return &_att_sp; }
```

Comment:
The formatting seems all off. Code style fail or style checker fail - I'm not sure.